frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/FrontendPluginFactory.java

```diff
@@ -1,35 +1,55 @@
 package com.github.eirslett.maven.plugins.frontend.lib;
 
+import com.github.eirslett.maven.plugins.frontend.lib.version.manager.VersionManagerCache;
+import com.github.eirslett.maven.plugins.frontend.lib.version.manager.VersionManagerRunner;
+import com.github.eirslett.maven.plugins.frontend.lib.version.manager.VersionManagerType;
+import com.github.eirslett.maven.plugins.frontend.lib.version.manager.VersionManagerLocator;
+import jdk.nashorn.internal.objects.Global;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.File;
+import java.util.Arrays;
 
 public final class FrontendPluginFactory {
 
+    private static final Logger logger = LoggerFactory.getLogger(FrontendPluginFactory.class);
+
     private static final Platform defaultPlatform = Platform.guess();
     private static final String DEFAULT_CACHE_PATH = "cache";
 
     private final File workingDirectory;
     private final File installDirectory;
     private final CacheResolver cacheResolver;
+    private final boolean useNodeVersionManager;
+    private String nodeVersion;
 
     public FrontendPluginFactory(File workingDirectory, File installDirectory){
         this(workingDirectory, installDirectory, getDefaultCacheResolver(installDirectory));
     }
 
     public FrontendPluginFactory(File workingDirectory, File installDirectory, CacheResolver cacheResolver){
+        this(workingDirectory, installDirectory, cacheResolver, false);
+    }
```

Comment:
Do we still use this constructor? Maybe we get rid of it and it'll be easy to reason about when merging upstream

Reply:
only called by the first constructor, will remove it